src/main/cpp/chassis/SwerveModule.cpp

```diff
@@ -183,13 +184,21 @@ void SwerveModule::RunCurrentState()
 /// @returns void
 void SwerveModule::SetDriveSpeed(units::velocity::meters_per_second_t speed)
 {
-    // Run Open Loop
-    m_activeState.speed = (abs(speed.to<double>() / m_maxSpeed.to<double>()) < 0.05) ? 0_mps : speed;
-    // convert mps to unitless rps by taking the speed and dividing by the circumference of the wheel
-    auto driveTarget = m_activeState.speed.to<double>() / (units::length::meter_t(m_wheelDiameter).to<double>() * numbers::pi);
-    auto percent = driveTarget / m_maxSpeed.to<double>();
-    DutyCycleOut out{percent};
-    m_driveTalon->SetControl(out);
+    if (m_velocityControlled)
+    {
+        m_activeState.speed = (abs(speed.to<double>() / m_maxSpeed.to<double>()) < 0.05) ? 0_mps : speed;
```

Comment:
This line is common between both branches of the if/else, so pull it out before the if

Reply:
I have made the fix